### ISSUE TITLE
Add niScope_wfmInfo support to NI-SCOPE

### ISSUE THREAD

**Issue by Fladolcetta** [ni/nimi-python#518]
This type is used all over the NI-SCOPE API. We need to support it in Python:

```
struct niScope_wfmInfo
{
ViReal64 absoluteInitialX;
ViReal64 relativeInitialX;
ViReal64 xIncrement;
ViInt32 actualSamples;
ViReal64 offset;
ViReal64 gain;
ViReal64 reserved1;
ViReal64 reserved2;
}; 
```

**Comment by marcoskirsch** [comment]
Fixed in #571

**Comment by marcoskirsch** [comment]
Reopened, and made issue specific to the NI-SCOPE type.
